package.json

```diff
@@ -407,26 +408,34 @@
     "contributor:add": "all-contributors add",
     "contributor:generate": "all-contributors generate",
     "contributor:check": "all-contributors check",
-    "test": "./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha -- -R spec --ui tdd './out/test/**/*.test.js'",
-    "test:local": "istanbul cover ./node_modules/mocha/bin/_mocha -- -R spec --ui tdd './out/test/**/*.test.js'",
+    "test": "nyc mocha --require ts-node/register --require source-map-support/register -- -R spec --ui tdd './test/**/*.test.ts'",
+    "test:local": "nyc mocha --require ts-node/register --require source-map-support/register -- -R spec --ui tdd './test/**/*.test.ts'",
```

Comment:
Perhaps we can remove this command `test:local` if `test` command can run successfully both in local and in the CI. 

Do you think it is better to have `mocha.opts` [link](https://mochajs.org/#-opts-path) file to specify the parameters? 

the `mocha.opts` will be like

```
--require ts-node/register
--require source-map-support/register
--reporter spec
--ui tdd
```
I feel that it will make the command defined here cleaner

Reply:
that sounds good